misfitapp/models.py

```diff
@@ -44,29 +96,41 @@ class Meta:
         unique_together = ('user', 'date')
 
     @classmethod
-    def create_from_misfit(cls, misfit, uid, start_date=datetime.date(2014,1,1), end_date=datetime.date.today()):
+    def import_from_misfit(cls, misfit, uid, update=False,
```

Comment:
@grokcode :smile: Good, me too. For a while I was calling it `create_or_update_from_misfit` :open_mouth: